integration-tests/common/test_common.go

```diff
@@ -0,0 +1,93 @@
+package common
```

Comment:
I didn't bring everything over from starknet for this file. Chances are a decent amount can be brought over mostly as is when writing the test.